modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/AbstractMultiNodeBenchmark.java

```diff
@@ -141,11 +146,9 @@ protected static void createTable(String tableName, List<String> columns, List<S
 
         createTableStatement += "\nZONE " + ZONE_NAME;
 
-        getAllFromCursor(
```

Comment:
Why it's removed?

Reply:
We switched to public synchronous SQL API. Why do we need to wait for something or iterate over the result set in this case?

Reply:
Yep, makes sense.